### PR TITLE
Unfused_Frozen_BatchNorm and Unfused_Upsample_with_two_input model and test data added

### PR DESCRIPTION
**Pull request by ashishkrshrivastava**

### CODE REVIEW

File: testdata/dnn/onnx/generate_onnx_models.py [hunk 1]
```diff
@@ -680,3 +680,30 @@ def forward(self, x):
 input = Variable(torch.randn(seq_len, batch, features))
 lstm = LSTM(features, hidden, batch, bidirectional=True)
 save_data_and_model("lstm_bidirectional", input, lstm)
+
+from torchvision.ops.misc import *
+n = 64
+model = FrozenBatchNorm2d(n)
+model.eval()
+input = Variable(torch.rand( 1, 64, 1, 1 ))
```
Comment (dkurt) [inline testdata/dnn/onnx/generate_onnx_models.py:688]: Please use different dimensions, like `[1, 4, 2, 3]`

File: testdata/dnn/onnx/generate_onnx_models.py [hunk 2]
```diff
@@ -680,3 +680,30 @@ def forward(self, x):
...
+
+    def forward(self, x, y):
+        x = self.conv1(x)
+        x = x.shape[-2:]
+        y = self.conv2(y)
+        y = F.interpolate(y, size=x, mode="nearest")
+        return y
+
+input_0 = Variable(torch.randn(1, 3, 20, 20))
+input_1 = Variable(torch.randn(1, 3, 10, 10))
```
Comment (dkurt) [inline testdata/dnn/onnx/generate_onnx_models.py:707]: Please use smaller but inputs which still are representative. To cover dimensions confusion, try to use different scales and different sizes.

File: testdata/dnn/onnx/generate_onnx_models.py [hunk 3]
```diff
@@ -680,3 +681,29 @@ def forward(self, x):
 input = Variable(torch.randn(seq_len, batch, features))
 lstm = LSTM(features, hidden, batch, bidirectional=True)
 save_data_and_model("lstm_bidirectional", input, lstm)
+
+from torchvision.ops.misc import *
+n = 3
```
Comment (dkurt) [inline testdata/dnn/onnx/generate_onnx_models.py:686]: unused variable?

File: testdata/dnn/onnx/generate_onnx_models.py [hunk 4]
```diff
@@ -680,3 +681,29 @@ def forward(self, x):
...
+
+    def forward(self, x, y):
+        x = self.conv1(x)
+        x = x.shape[-2:]
+        y = self.conv2(y)
+        y = F.interpolate(y, size=x, mode="nearest")
+        return y
+
+input_0 = Variable(torch.randn(1, 3, 2, 4))
+input_1 = Variable(torch.randn(1, 3, 1, 2))
```
Comment (dkurt) [inline testdata/dnn/onnx/generate_onnx_models.py:707]: Please test also different scales:
```python
input_0 = Variable(torch.randn(1, 3, 3, 4))
input_1 = Variable(torch.randn(1, 3, 1, 2))
```

File: testdata/dnn/onnx/generate_onnx_models.py (hunk 3 above)
Comment (ashishkrshrivastava) [reply testdata/dnn/onnx/generate_onnx_models.py:686]: No, used in the next line. 
`model = FrozenBatchNorm2d(n)`

File: testdata/dnn/onnx/generate_onnx_models.py (hunk 1 above)
Comment (ashishkrshrivastava) [reply testdata/dnn/onnx/generate_onnx_models.py]: Done

File: testdata/dnn/onnx/generate_onnx_models.py (hunk 4 above)
Comment (ashishkrshrivastava) [reply testdata/dnn/onnx/generate_onnx_models.py]: Done

File: testdata/dnn/onnx/generate_onnx_models.py (hunk 3 above)
Comment (dkurt) [reply testdata/dnn/onnx/generate_onnx_models.py:686]: Sorry, my mistake!

File: testdata/dnn/onnx/generate_onnx_models.py [hunk 5]
```diff
@@ -740,3 +741,30 @@ def forward(self, x):
...
+    def __init__(self):
+        super(UpsampleUnfusedTwoInput, self).__init__()
+        self.conv1 = nn.Conv2d(3, 3, kernel_size=1, stride=1, padding=0)
+        self.conv2 = nn.Conv2d(3, 3, kernel_size=1, stride=1, padding=0)
+
+    def forward(self, x, y):
+        x = self.conv1(x)
+        x = x.shape[-2:]
+        y = self.conv2(y)
+        y = F.interpolate(y, size=x, mode="nearest")
```
Comment (dkurt) [inline testdata/dnn/onnx/generate_onnx_models.py:763]: What if we will swap x and y so first input will be source but second used for shapes? Our PR works in that scenario?

File: testdata/dnn/onnx/generate_onnx_models.py [hunk 6]
```diff
@@ -742,6 +743,7 @@ def forward(self, x):
 x = Variable(torch.randn(1, 2, 3, 4))
 save_data_and_model("reduceL2_subgraph", x, model)
 
+<<<<<<< HEAD
```
Comment (dkurt) [inline testdata/dnn/onnx/generate_onnx_models.py:746]: Please resolve rebase artifacts

Comment (alalek) [reply testdata/dnn/onnx/generate_onnx_models.py:746]: BTW, consider enabling diff3 conflict style:
- https://stackoverflow.com/questions/27417656/should-diff3-be-default-conflictstyle-on-git
- `git config --global merge.conflictstyle diff3`

Comment (dkurt) [reply testdata/dnn/onnx/generate_onnx_models.py:746]: /cc @ashishkrshrivastava

File: testdata/dnn/onnx/generate_onnx_models.py (hunk 5 above)
Comment (ashishkrshrivastava) [reply testdata/dnn/onnx/generate_onnx_models.py:799]: As there is no longer subgraph fusion for `UpsampleUnfusedTwoInputs` subgraph. Each node for that subgraph is enabled. so it doesn't matter swapping x and y.

File: testdata/dnn/onnx/generate_onnx_models.py (hunk 6 above)
Comment (ashishkrshrivastava) [reply testdata/dnn/onnx/generate_onnx_models.py:746]: Oh, I am sorry. I was working on another PR for last two days. So couldn't notice. 
I will correct above artifacts.

Comment (ashishkrshrivastava) [reply testdata/dnn/onnx/generate_onnx_models.py]: Done !

Comment (ashishkrshrivastava) [reply testdata/dnn/onnx/generate_onnx_models.py]: > BTW, consider enabling diff3 conflict style:
> 
> * https://stackoverflow.com/questions/27417656/should-diff3-be-default-conflictstyle-on-git
> * `git config --global merge.conflictstyle diff3`

@alalek Done! thanks

**Pull request merged**